src/utils/context.ts

```diff
@@ -22,13 +24,53 @@ export const DefaultCowContext = {
  * @class Context
  * @implements {Required<CowContext>}
  */
-export class Context implements Required<CowContext> {
+export class Context implements Partial<CowContext> {
   private context: CowContext
+  #chainId: ChainId
```

Comment:
What's the reason for using TS private (`private`) for context and JS private (`#`) for chainId?

Reply:
good catch! when I updated the chainId accessor didn't change the context one.  This was to enforce runtime access outside of TS